qt/scientific_interfaces/Indirect/IndirectDataAnalysis.cpp

```diff
@@ -116,7 +116,7 @@ void IndirectDataAnalysis::loadSettings() {
  */
 void IndirectDataAnalysis::tabChanged(int) {}
 
-std::string IndirectDataAnalysis::documentationPage() const { return "Indirect Data Analysis"; }
+std::string IndirectDataAnalysis::documentationPage() const { return "Inelastic Data Analysis"; }
```

Comment:
Out of curiosity, while the title of the interface is now inelastic, the menu item is still in Indirect, is this expected? Was it not the plan to rename also the menu entry?

Reply:
Good spot. @Harrietbrown was the menu item going to change at some point?

Reply:
The plan is to eventualy have access to the interface in both the direct and indirect menus.